### PR TITLE
Made markdown ready for pdf generation.

### PR DESCRIPTION
**Pull request by avandeursen**
Adjusted the markdown and Makefile so that `pandoc` can generate pdf (via latex) as well.

Main modifications include standardizing image, table, and figure handling, as well as handling preformatted code (mostly replacing html with real markdown).

### CODE REVIEW

File: chapters/angulardart/index.md [hunk 1]
```diff
@@ -196,7 +194,7 @@ The functional capabilities define what AngularDart does. AngularDart is a port
 We derived the following functions in AngularDart:
 
 | Function | In component |
-|----------|--------|
+|----------------------------------------------------|------------|
```
Comment (rogierslag) [inline chapters/angulardart/index.md:197]: does Pandoc depend on this length being correct? GFM does not care

File: chapters/angulardart/index.md [hunk 2]
```diff
@@ -257,7 +254,8 @@ Connectors are the pieces of code of the architecture that link the elements, or
 #### External entities
 AngularDart has "other systems, software programs, hardware devices or other entities with which the system interacts" (Rozanski and Woods, p. 272), or external entities. These can be extracted from the Context view image below.
 
-![Context view with logos](images/Context_view_with_logos.png "Context view with logos")
+![](images/Context_view_with_logos.png)
+_Context view with logos_
```
Comment (rogierslag) [inline chapters/angulardart/index.md:258]: Might be preferably to have these both below and as alt text for accessibility for screen readers and crawlers

**Comment by rogierslag** [comment]
Looks good, just had two questions :+1:

File: chapters/angulardart/index.md (hunk 1 above)
Comment (avandeursen) [inline chapters/angulardart/index.md:197]: I sort of reverse engineered this, but I don't remember where I read about it. Probably inspired by
http://pandoc.org/README.html#tables

But indeed, when creating pdf or epub, pandoc needs to select a column width. It uses various heuristics, which can be influenced by the actual column widths in number of characters in headers like these.

This is indeed ignored by GFM / Jekyll.

File: chapters/angulardart/index.md (hunk 2 above)
Comment (avandeursen) [inline chapters/angulardart/index.md:258]: You are right.

The tricky thing is that to get pandoc with automated numbering and cross referencing right, having _just_ the alt text is not enough. So this leads to some not-so-nice duplication of captions.

I've updated the captions (with a simple sublime edit macro) -- will push the commit shortly.

**Comment by rogierslag** [comment]
Ready for merge if you ask me

**Comment by avandeursen** [comment]
:+1:, :shipit:

**Comment by avandeursen** [comment]
After that I'll create a release with the final pdf + epub.

**Pull request merged**